update.yaml

```diff
@@ -11,6 +11,9 @@ STEPS:
   #  use_title: 1
   #  use_date: 1
 
+  - action: check_for_placeholders
```

Comment:
Let's give this a more descriptive name that reflects the action, e.g. embed_lessons_media